### PR TITLE
build mft-kernel binary debian package

### PR DESCRIPTION
**Pull request by lguohan**

### CODE REVIEW

File: src/mft/build.sh [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+MFT_NAME=mft-4.5.0-31-x86_64-deb
+MFT_TGZ=${MFT_NAME}.tgz
+MFT_KERNEL_DEB=kernel-mft-dkms_4.5.0-31_all.deb
+KERNELVER=3.16.0-4-amd64
+
+[ -e $MFT_TGZ ] || wget http://www.mellanox.com/downloads/MFT/${MFT_TGZ}
+tar xzf $MFT_TGZ
+pushd $MFT_NAME/SDEBS
+dpkg -i $MFT_KERNEL_DEB
+TARBALL_PATH=$(dkms mkdriverdisk kernel-mft-dkms/4.5.0 -d ubuntu -k ${KERNELVER} --media tar | grep "Disk image location" | cut -d':' -f2)
```
Comment (marian-pritsak) [inline src/mft/build.sh]: dkms mkdriverdisk command creates debian package with architecture x86_64, which yields an error during instalation due to arch mismatch
You can add option -a all to set architecture type to all to allow any arch

Comment (marian-pritsak) [inline src/mft/build.sh]: `dkms mkdriverdisk` command creates debian package with architecture x86_64, which yields an error during instalation due to arch mismatch
You can add option `-a all` to set architecture type to all to allow any arch

File: src/mft/build.sh [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+##
+## USAGE:
+##   ./build.sh
+
+MFT_NAME=mft-4.5.0-31-x86_64-deb
+MFT_TGZ=${MFT_NAME}.tgz
+MFT_KERNEL_DEB=kernel-mft-dkms_4.5.0-31_all.deb
+KERNELVER=3.16.0-4-amd64
+
+[ -e $MFT_TGZ ] || wget http://www.mellanox.com/downloads/MFT/${MFT_TGZ}
```
Comment (qiluo-msft) [inline src/mft/build.sh]: wget -N: to detect remote file content updating.

**Comment by lguohan** [comment]
updated.

**Pull request merged**